spec/system/support_interface/reinstate_offer_to_declined_course_choice_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+RSpec.feature 'Reinstate offer to a declined course choice' do
+  include DfESignInHelpers
+
+  scenario 'Support user can reverse a course choice that has accidently been declined' do
+    given_i_am_a_support_user
+    and_the_reinstate_offer_feature_flag_is_on
+    and_there_is_a_submitted_application_in_the_system_with_a_declined_offer
+    and_i_visit_the_support_page
+
+    when_i_click_on_an_application
+    and_i_am_on_the_correct_application_page
+    then_i_see_the_declined_course_choice
+
+    when_i_click_on_the_reinstate_offer_link
+    then_i_see_the_reinstate_offer_page
+    and_i_click_continue
```

Comment:
when_i_click_continue
then_i_am_told_to_confirm_i_have_followed_the_guidance

it's always in this order:

`-> given -> when -> then `

with optional `and`'s

affirmative actions i.e. clicking on something are always part of the given/when portion